### PR TITLE
Converting medication request

### PR DESCRIPTION
**Pull request by ItayGoren**

### CODE REVIEW

File: org.hl7.fhir.convertors/src/test/java/org/hl7/fhir/convertors/conv10_40/MedicationRequest10_40Test.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            "   \"method\": {\"coding\": [{\"system\": \"urn:oid:1.2.840.114350.1.13.0.1.7.4.798268.8600\","+
+            "      \"code\": \"11\","+
+            "      \"display\": \"Take\"}],"+
+            "    \"text\": \"Take\"},"+
+            "   \"doseAndRate\": [{\"doseQuantity\": {\"value\": 10.0,"+
+            "      \"unit\": \"mg\","+
+            "      \"system\": \"http://unitsofmeasure.org\","+
+            "      \"code\": \"mg\"}}]}],"+
+            " \"dispenseRequest\": {\"validityPeriod\": {\"start\": \"2016-11-15T00:00:00Z\","+
+            "   \"end\": \"2016-11-23T00:00:00Z\"}}}";
```
Comment (markiantorno) [inline org.hl7.fhir.convertors/src/test/java/org/hl7/fhir/convertors/conv10_40/MedicationRequest10_40Test.java]: please put two strings into separate *.json resource files, and then load them with the class loader

File: .editorconfig [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [*.java]
 charset = utf-8
 indent_style = space
-indent_size = 2
+indent_size = 4
```
Comment (markiantorno) [inline .editorconfig]: Please don't change or commit the editor config file.

Comment (ItayGoren) [reply .editorconfig]: my bad, fixed it

File: org.hl7.fhir.convertors/src/test/java/org/hl7/fhir/convertors/conv10_40/MedicationRequest10_40Test.java (hunk 1 above)
Comment (ItayGoren) [reply org.hl7.fhir.convertors/src/test/java/org/hl7/fhir/convertors/conv10_40/MedicationRequest10_40Test.java]: fixed it - looks better now
thanks

**Review by grahamegrieve — COMMENTED** [review]
ok

**Pull request merged**